README.md

```diff
@@ -1,39 +1,51 @@
 
 # Offliner
-Offliner is a library for [service workers](http://www.html5rocks.com/en/tutorials/service-worker/introduction/?redirect_from_locale=ja) providing and lifecycle for your web applications.
+Offliner is a library that implements a lifecycle for web applications utilizing [service workers](http://www.html5rocks.com/en/tutorials/service-worker/introduction/?redirect_from_locale=ja). The primary goal of offliner is to enable developers to provide an offline experience for their applications and easily dispatch updates to users as needed.
 
-## Remember
-
-This technology can be used in the following browsers:
+#### Compatibility
+Service workers are still a very new technology, which means the progress of implementation varies across platforms. As of June 25th, 2015, this technology can be used in the following browsers:
   * [Chrome](https://www.google.com/chrome/)
   * [Firefox Nightly](https://nightly.mozilla.org/)
   * [Firefox OS](https://developer.mozilla.org/en-US/Firefox_OS/Installing_on_a_mobile_device)
 
-For Nightly and Firefox OS you need to [configure some things](https://blog.wanderview.com/blog/2015/03/24/service-workers-in-firefox-nightly/) before using.
-
-Pay attention to the [progress of Service Workers implementations](https://jakearchibald.github.io/isserviceworkerready/) and [review the W3C draft](http://www.w3.org/TR/2014/WD-service-workers-20141118/) for more information.
-
-## The Web App lifecycle
-
-When you think about making your application to work offline you realize your web application has a life-cycle: you need it to get **installed**, then to be **served** and from time to time, to be **updated**. Offliner is a plugable framework to implement this life-cycle.
+For Firefox OS you will need to [configure some things](https://blog.wanderview.com/blog/2015/03/24/service-workers-in-firefox-nightly/) before using.
```

Comment:
I think now is even more tricky. Can try to update this link with up-to-date information?